### PR TITLE
Adding QRH (Quick Reference Handbook)

### PR DESCRIPTION
**Pull request by messorstruktor**
Added several situations from QRH to checklists. I will add more situations later. Now there are:
-GROUND EVACUATION
-RUDDER JAM
-STABILIZER JAM

Fixes #[issue_no]

**Summary of Changes**

**Screenshots (if necessary)**

**References**

**Additional context**

Discord username (if different from GitHub): FBI#7771

### CODE REVIEW

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml [hunk 1]
```diff
@@ -561,11 +561,37 @@
...
 			</Page>
 		</Step>
-
-
-
-
+		<!-- EMERGENCY -->
+		<Step ChecklistStepId="LANDING_GATE">
+        	<Page SubjectTT="TT:ON GROUND EVACUATION (EMERGENCY)">
+           		<Checkpoint ReferenceId="A320_EMERGENCY_PARKING_BRAKE"/>
```
Comment (DoToAdventures) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:567]: Indentation

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml [hunk 2]
```diff
@@ -561,11 +561,37 @@
...
+            	<Checkpoint ReferenceId="A320_EMERGENCY_ATC"/>
+            	<Checkpoint ReferenceId="A320_EMERGENCY_ENGINE_MASTER"/>
+            	<Checkpoint ReferenceId="A320_EMERGENCY_CABINCREW"/>
+            	<Checkpoint ReferenceId="A320_EMERGENCY_FIRE_ALERT"/>
+            	<Checkpoint ReferenceId="A320_EMERGENCY_AGENTS"/>
+            	<Checkpoint ReferenceId="A320_EMERGENCY_EVACUATION"/>
+        	</Page>
+            <Page SubjectTT="TT:RUDDER JAM (EMERGENCY)">
+               <Block SubjectTT="FOR APPROACH">
+                  	<Checkpoint ReferenceId="A320_EMERGENCY_AUTOBRK"/>
```
Comment (DoToAdventures) [inline A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:577]: Indentation

**Review by DoToAdventures — COMMENTED** [review]
It would be good to implement highlighting for some of the instruments. You can view the Library.xml for examples on this is done for instruments where I've already done it, for example the engine master switches are highlighted by adding
```
<Instrument Id="ENGINE_Switch_Master1" />
<Instrument Id="ENGINE_Switch_Master2" />
```
to the checkpoint A320_EMERGENCY_ENGINE_MASTER

File: A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml (hunk 2 above)
Comment (messorstruktor) [reply A32NX/SimObjects/AirPlanes/Asobo_A320_NEO/Checklist/Airbus_A320neo_Checklist.xml:577]: Okay, I'll add object highlighting.

**Comment by messorstruktor** [comment]
All objects that can be highlighted are now highlighted.

**Comment by AdenFlorian** [comment]
@messorstruktor did you mean to mark this as a draft? are you still working on some changes to this?

**Comment by AdenFlorian** [comment]
tested and looks fine to merge to me, some of the items have highlighting, more can be added later, thanks!

**Pull request merged**